website/project/views/drafts.py

```diff
@@ -0,0 +1,217 @@
+from flask import request, redirect
+import httplib as http
+from dateutil.parser import parse as parse_date
+import functools
+
+from modularodm import Q
+from modularodm.exceptions import ValidationValueError
+
+from framework import status
```

Comment:
You're going to be getting a note about this from flake soon enough, but this import is unused.

```
inv flake8
./website/project/views/drafts.py:9:1: F401 'status' imported but unused
```